newsfragments/198.bugfix

```diff
@@ -0,0 +1 @@
+HDF5 / NeXus: correctly pass the mask with get_static_mask() not get_mask(); includes bug fix where Eiger data report incorrect data_size. 
```

Comment:
It might be worth rewording this with a view to how it would be understood by the average user reading the release notes. I.e. references to `get_static_mask()` and `get_mask()` are implementation details which they almost certainly don't care about.

Reply:
Reworded